group_vars/all/ip_addresses.yml

```diff
@@ -22,6 +22,14 @@ ip_addresses:
       address: '20.67.122.56'
       netmask: '/32'
       desc: 'UMCG firewalls / core routers'
+    internal:
```

Comment:
If this is the official name for this network range I do not mind, but otherwise `internal` is too generic; there will be many other internal network ranges.

Reply:
Currently there is no confusion, as there is only one internal network. And I had to named it something. If we actually need to add more later, then we can simply change it  `internal1`, `internal2` ... I don't see UMCG network named `internal` any less generic as f.e. network named `net1`, `net2` ... and the name does describe the type of network, so as does description.

Reply:
I changed to `internal1`.